<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Historique</class>
 <widget class="QWidget" name="Historique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Historique</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 245, 245);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="groupBoxFiltre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Filtres</string>
   </property>
   <widget class="QPushButton" name="pushButtonAppliquer">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Appliquer</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>431</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutHeureDate">
     <item>
      <widget class="QLabel" name="HeureDate">
       <property name="text">
        <string>Heure et date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDe">
       <property name="text">
        <string>De</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeDebut">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(253, 245, 245);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="displayFormat">
        <string>dd-MM-yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelA">
       <property name="text">
        <string>Ã </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeFin">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(253, 245, 245);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="displayFormat">
        <string>dd-MM-yyyy HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutAgents">
     <item>
      <widget class="QLabel" name="Agents">
       <property name="text">
        <string>Agents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxAgents">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentText">
        <string>Tous les agents</string>
       </property>
       <item>
        <property name="text">
         <string>Tous les agents</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutRonde">
     <item>
      <widget class="QLabel" name="Ronde">
       <property name="text">
        <string>Ronde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxRonde">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(253, 245, 245);
color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>Toutes les rondes</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>731</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
